src/MainPage.js

```diff
@@ -2,33 +2,40 @@ import { useEffect } from 'react'
 import styled from 'styled-components/macro'
 import Form from './components/Form'
 
-
-function MainPage({setUserPlace, errorMessage, isCountyDataLoaded, resetSearch}) {   
-    
-    useEffect(() => {isCountyDataLoaded && resetSearch()}, [])
+function MainPage({
+    setUserPlace,
+    errorMessage,
+    isDataLoading,
+    isCountyDataLoaded,
+    resetSearch,
+    setSearchOrigin,
+}) {
+    useEffect(() => {
+        isCountyDataLoaded && resetSearch()
+    }, [])
 
     return (
         <MainPageStyled>
             <h1>Bin ich in einem Covid-19 Hotspot?</h1>
-            <Form 
+            <Form
                 setUserPlace={setUserPlace}
                 errorMessage={errorMessage}
-                >
-            </Form>
-        </MainPageStyled> 
+                isDataLoading={isDataLoading}
+                setSearchOrigin={setSearchOrigin}
+            ></Form>
```

Comment:
You don't embed anything in your form tag. So you can close it directly without a closing tag.